### DISCUSSION TITLE
Make possible same recipient in different accounts

### DISCUSSION THREAD

**Discussion by 08c15** [anonaddy/anonaddy#420] [Feature Requests]
If I try to add a recipient to my account, that is already present in another account, it is denied.
Would greatly appreciate, if this limitation would be removed.
(Usecase: For example some aliases prom "Father's" anonaddy account shall be received by the son's email as well as some others from "mother's" anonaddy account)

**Comment by willbrowningme** [comment]
Thanks for the suggestion, I'm afraid this is unlikely to be implemented as it would allow people to easily circumvent bandwidth limits for free accounts by making multiple accounts.

**Reply by 08c15** [reply]
@willbrowningme : Thank you very much for your reply.
Perhaps it would be a solution, to make this possible for paid accounts only ?
